### PR TITLE
[W5.7][T13-2]Ngeow Xiu Qi

### PR DESCRIPTION
**Pull request by NgeowXiuQi**
Added a search by phone number feature.

### CODE REVIEW

File: test/java/seedu/addressbook/commands/SearchNumberCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  @Test
+  public void execute() {
+
+    //number not belonging to anyone: not matched
+    assertSearchCommandBehavior("98472637", Collections.emptyList());
+
+    //number belonging to one user (Amy): matched
+    assertSearchCommandBehavior("91119111", Arrays.asList(td.amy));
+
```
Comment (IamRENCE) [inline test/java/seedu/addressbook/commands/SearchNumberCommandTest.java:30]: Good to follow code style and remove any unnecessary newline characters

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (DanKhoo) [inline src/seedu/addressbook/parser/Parser.java:14]: Try to disable IntelliJ to use wildcard imports and include each and every import individually

File: test/java/seedu/addressbook/commands/SearchNumberCommandTest.java (hunk 1 above)
Comment (IamRENCE) [reply test/java/seedu/addressbook/commands/SearchNumberCommandTest.java]: d

File: src/seedu/addressbook/parser/Parser.java (hunk 2 above)
Comment (xhsien) [reply src/seedu/addressbook/parser/Parser.java:14]: Agree with @DanKhoo. It is better to import each class individually.

**Review by xhsien — APPROVED** [review]
Good work, @NgeowXiuQi . There are no issues on oop standards, and you did a good job on updating the user guide and tests 👍

I also see that you have the tests for `isAnyNull`. That is the work from previous week and shouldn't be added here. For the coding standards, remember to import every class.

Please close this PR after reading this comment.

**Pull request closed without merging**